heron/packing/tests/java/com/twitter/heron/packing/AssertPacking.java

```diff
@@ -90,11 +90,10 @@ public static void assertNumInstances(Set<PackingPlan.ContainerPlan> containerPl
   public static void assertContainerRam(Set<PackingPlan.ContainerPlan> containerPlans,
                                         long maxRamforResources) {
     for (PackingPlan.ContainerPlan containerPlan : containerPlans) {
-      long containerRam = 0;
-      for (PackingPlan.InstancePlan instancePlan : containerPlan.getInstances()) {
-        containerRam += instancePlan.getResource().getRam();
-      }
-      Assert.assertTrue(containerRam <= maxRamforResources);
+      Assert.assertTrue("Container with id: " + containerPlan.getId() + " requires more RAM " + "("
```

Comment:
@billonahill Why is that important? Is there a performance difference? My understanding is that this message will be printed only when a failure happens.